ui/src/main/resources/GoogleApps/GoogleAppsConfigSheet.vm

```diff
@@ -6,108 +6,145 @@ $xwiki.ssx.use('GoogleApps.GoogleAppsConfigClass')
-<input type="checkbox" name="scope_identity" disabled="true" checked>#t("scope_identity")</label> ##
-<label title="#t('checkbox_mandatory')"> ##
-<input type="checkbox" name="scope_email"    disabled="true" checked>#t("scope_email")</label> ##
-<label title="#t('checkbox_mandatory')"> ##
-<input type="checkbox" name="scope_drive"                           >#t("scope_drive")</label> ##
-<label title="#t('checkbox_notyetdone')"> ##
-<input type="checkbox" name="scope_avatar"                          >#t("scope_avatar")</label> ##
-$configDoc.display("scope", "hidden")
-
-; <label for="${prefix}_appname">#t("appname")</label> ##
-<span class="xHint">#t("appname.hint")</span>
-: $configDoc.display("appname", 'edit', $obj)
-
-</fieldset>
-
-<fieldset>
-<legend>#t("loginbehaviour")</legend>
-
-; <label for="${prefix}_domain">#t("domain")</label> ##
-<span class="xHint">#t("domain.hint")</span>
-: $configDoc.display("domain", 'edit', $obj)<br> ##
-<span id="googleapps-domain-livehint" class="xHint">&nbsp;</span>
-
-; $configDoc.display("useCookies", 'edit', $obj) <label for="${prefix}_useCookies">#t("useCookies")</label> ##
-<span class="xHint">#t("useCookies.hint")</span>
-
-:; $configDoc.display("skipLoginPage", 'edit', $obj) ##
-<label for="${prefix}_skipLoginPage">#t("skipLoginPage")</label> ##
-<span class="xHint">#t("skipLoginPage.hint")</span>
-
-:; $configDoc.display("authWithCookies", 'edit', $obj) ##
-<label for="${prefix}_authWithCookies">#t("authWithCookies")</label> ##
-<span class="xHint">#t("authWithCookies.hint")</span>
-
-:; <label for="${prefix}_cookiesTTL">#t("cookiesTTL")</label> ##
-<span class="xHint">#t("cookiesTTL.hint")</span>
-:: $configDoc.display("cookiesTTL", 'edit', $obj)
-</fieldset>
-
-
-## Hidden form elements
-#set($params="editor=${escapetool.url(${editor})}&amp;section=${escapetool.url(${section})}")
-#set($params="${params}&amp;space=${escapetool.url(${currentSpace})}")
-#set($continueURL=$xwiki.getURL($currentDoc, 'admin', $params))
-<input type="hidden" name="form_token" value="$!{services.csrf.getToken()}" />
-<input type="hidden" name="xcontinue" value="${continueURL}" />
-<input type="hidden" name="xredirect" value="${continueURL}" />
-
-## submit
-<div class="bottombuttons">
-<p class="admin-buttons">
-<span class="buttonwrapper"><input class="button" type="submit" name="formactionsac" ##
-value="$services.localization.render('admin.save')" /></span>
-</p>
-</div>
+  <dl>
+    <dt>#displayInput ('activate') <label for="${propNamePrefix}_activate">#t ('activate')</label></dt>
+    <dd></dd>
+  </dl>
+
+  <fieldset>
+    <legend>#t ('communicate')</legend>
+
+    <dl>
+      <dt><span class="xHint">
+        #tp ('communicate.hint',  ['[[', '>>https://accounts.google.com/ManageDomains]]'])<br>
+        #tp ('communicate.hint2', ['[[', '>>https://store.xwiki.com/xwiki/bin/view/Extension/GoogleAppsIntegration#installation]]'])</span>
+      </dt>
+      <dd></dd>
+    </dl>
```

Comment:
Why do you create a definition list for each ``dt, dd`` pair?

Reply:
This was inherited from earlier code and seems indeed not necessary.
removed.
Thanks.